tycho-its/projects/target.maven.eclipse-feature/target-platform/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```

Comment:
project files are usually not required to be included in integration tests.

Reply:
> project files are usually not required to be included in integration tests.

Thanks.  Removed